src/gui/static/src/app/utils/semver.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Compares two SemVer versions, returns true if 'from' is smaller than 'to'.
+ * Special cases with 'rc' suffix are described in spec file.
+ *
+ * @returns {boolean}
+ * @param from
+ * @param to
+ */
+export function shouldUpgradeVersion(from: string, to: string): boolean {
+  const fromParts = to.split('.');
+  const toSplit = from.split('-');
+  const toParts = toSplit[0].split('.');
+
+  for (let i = 0; i < 3; i++) {
+    const fromNumber = Number(fromParts[i]);
+    const toNumber = Number(toParts[i]);
+
+    if (fromNumber > toNumber) {
+      return true;
```

Comment:
shudnt this be `false` and the other one `true`

Reply:
I forgot to rename the variables